src/sentry/static/sentry/app/stores/organizationStore.jsx

```diff
@@ -9,20 +11,36 @@ const OrganizationStore = Reflux.createStore({
     this.listenTo(OrganizationActions.update, this.onUpdate);
     this.listenTo(OrganizationActions.fetchOrg, this.reset);
     this.listenTo(OrganizationActions.fetchOrgError, this.onFetchOrgError);
+
+    // mark the store as dirty if projects or teams change
+    this.listenTo(ProjectActions.createSuccess, this.onProjectOrTeamChange);
+    this.listenTo(ProjectActions.updateSuccess, this.onProjectOrTeamChange);
+    this.listenTo(ProjectActions.loadStatsSuccess, this.onProjectOrTeamChange);
+    this.listenTo(ProjectActions.changeSlug, this.onProjectOrTeamChange);
+    this.listenTo(ProjectActions.addTeamSuccess, this.onProjectOrTeamChange);
+    this.listenTo(ProjectActions.removeTeamSuccess, this.onProjectOrTeamChange);
+
+    this.listenTo(TeamActions.updateSuccess, this.onProjectOrTeamChange);
+    this.listenTo(TeamActions.fetchDetailsSuccess, this.onProjectOrTeamChange);
```

Comment:
Do we need to mark as dirty for this action?

Reply:
I was unsure as this action was hooked up to the `onUpdateSuccess` method inside of `TeamStore` so I just wanted to err on the side of safety. Could this action change `TeamStore` meaningfully? Or is it similar to the `loadStatsSuccess` in `ProjectStore` where it is just populating details/stats for a team?

Reply:
Ah I see, it looks like the fetch and update actions call `onUpdateSuccess()` - I think it's safe to ignore the `fetchDetailsSuccess` action, we're only interested in marking the store as dirty on writes. Otherwise, fetching team details will result in a dirty org store.